data/random-teams.ts

```diff
@@ -172,7 +172,7 @@ export class RandomTeams {
 			Ground: (movePool, moves, abilities, types, counter) => !counter.get('Ground'),
 			Ice: (movePool, moves, abilities, types, counter) => (movePool.includes('freezedry') || !counter.get('Ice')),
 			Normal: (movePool, moves, abilities, types, counter) => {
-				if (movePool.includes('boomburst')) return true;
+				if (movePool.includes('boomburst') || movePool.includes('hypervoice')) return true;
```

Comment:
```suggestion
				return (movePool.includes('boomburst') || movePool.includes('hypervoice'));
```

Reply:
Forcing Normal STAB on Farigiraf's sets with Future Sight is no longer needed, since Hyper Voice is forced.